docs/details/statistics.dox

```diff
@@ -62,6 +62,21 @@ Find the correlation coefficient of values in the input
 
 \copydoc batch_detail_stat
 
+========================================================
+\defgroup stat_func_topk topk
+
+\ingroup basicstats_mat
+
+This function returns the top k values along a given dimension of the input
+array. The indices along with their values are returned. If the input is a
+multi-dimensional array, the indices will be the index of the value in that
+dimension. Order of duplicate values are not preserved. This function is
+optimized for small values of k.
```

Comment:
It would be nice if we can add some sort of guideline as to from which value of k does the performance deteriorate.

Reply:
This would be different for different devices and implementations.